scripts/quests/jeuno/Wings_of_Gold.lua

```diff
@@ -0,0 +1,100 @@
+-----------------------------------
+-- Wings of GOld
+-----------------------------------
+-- Log ID: 3, Quest ID: 60
+-- Brutus                      : !gotoid 17776652
```

Comment:
I'd prefer you use !pos values here, since NPC IDs can shift

Reply:
Sure thing, swapped out Brutus id for pos